### ISSUE TITLE
php artisan snipeit:backup error

### ISSUE THREAD

**Issue by CoventryIT** [snipe/snipe-it#7122]
#### Please confirm you have done the following before posting your bug report:

- [Yes ] I have enabled debug mode 
- [Yes ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**

When I open CMD as an administrator on my windows IIS server and change directories to C:\inetpub\wwwroot\snipe-it , Then try and run php artisan snipeit:backup I get the following error. Starting backup...

In BackupDestination.php line 147:

  Parse error: syntax error, unexpected '?'

**To Reproduce**
1. Go to 'CMD.exe'
2. Click on 'Run as Administrator'
3. change directories to C:\inetpub\wwwroot\snipe-it , Then try and run php artisan snipeit:backup"

**Expected behavior**
The backup process should start and pump a copy of my database in C:\inetpub\wwwroot\snipe-it\storage\app\backups

**Screenshots**
![image](https://user-images.githubusercontent.com/48068571/58961448-1f730380-8777-11e9-9068-5c0136f462fb.png)

**Server (please complete the following information):**
 - Snipe-IT Version 
 - OS: [e.g. Ubuntu, CentOS]
Windows Server 2012 R2
 - Web Server: [e.g. Apache, IIS]
IIS
 - PHP Version 
7.2.15

**Desktop (please complete the following information):**
Windows 10
Chrome
Version 74.0.3729.169 (Official Build) (64-bit)

**Smartphone (please complete the following information):**

**Error Messages**
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console.
- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
- Include any additional information you can find in `storage/logs` and your webserver's logs.

**Additional context**
- Is this a fresh install or an upgrade? 
upgrade
- What OS and web server you're running Snipe-IT on
Windows 2012 R2 IIS
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
Docker
- Include what you've done so far in the installation, and if you got any error messages along the way.
Upgrade was successful but unable to to run the backup command using command prompt. However I can still create backups using the backups section accessible on the web console.
- Indicate whether or not you've manually edited any data directly in the database
No

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

**Comment by CoventryIT** [comment]
By removing the "?" on line 147,152, 59  the backup

**Comment by jmartins1** [comment]
CoventryIT I'm having the same issue. This is what I did to work around the issue.

I'm running my installation on a WAMP server. I had to re-run the composer application before the following steps would function without error.

Commands starting with PHP run as followed:

Change the directory to the php version you are running then run your command(s). Ensure to provide the explicit path to your snipe-it folder. 

EXAMPLE:
CD "C:\wamp\bin\php\php7.3.1"
php "C:\wamp\www\snipe-it\artisan" snipeit:expected-checkin


EXAMPLE: Does not start with PHP
CD C:\wamp\www\snipe-it
composer install --no-dev --prefer-source
